src/NServiceBus.Host32/NServiceBus.Host32.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net452</TargetFramework>
+    <DebugType>full</DebugType>
+    <PlatformTarget>x86</PlatformTarget>
+    <RootNamespace>NServiceBus.Hosting.Windows</RootNamespace>
+    <ApplicationIcon>..\NServiceBus.Hosting.Windows\NServiceBus.ico</ApplicationIcon>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
+    <Description>Functionality for generically hosting NServiceBus processes</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Topshelf" HintPath="..\..\lib\Topshelf\Topshelf.dll" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommonServiceLocator" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Fody" Version="2.2.0" PrivateAssets="All" />
+    <PackageReference Include="log4net" Version="1.2.10" PrivateAssets="All" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0-beta0011, 8.0.0)" />
+    <PackageReference Include="Obsolete.Fody" Version="4.3.2" PrivateAssets="All" />
+    <PackageReference Include="Particular.CodeRules" Version="0.1.1" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\NServiceBus.Hosting.Windows\**\*.cs" Exclude="..\NServiceBus.Hosting.Windows\obj\**\*.cs" />
```

Comment:
What is up with this exclusion? Why is this needed? Obj would be excluded by default if I'm not mistaken?

Reply:
This because it's adding all the files from the other folder, and no, obj is not excluded by default.